migrations/versions/ce01ada6ac80_.py

```diff
@@ -0,0 +1,54 @@
+"""
+1.3.0 - Add biospecimen_diagnosis association table
+
+Revision ID: ce01ada6ac80
+Revises: 088f00b10226
+Create Date: 2018-07-25 11:55:35.583312
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import dataservice
+
+# revision identifiers, used by Alembic.
+revision = 'ce01ada6ac80'
+down_revision = '088f00b10226'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('biospecimen_diagnosis',
+                    sa.Column('uuid', postgresql.UUID(), nullable=True),
+                    sa.Column('created_at', sa.DateTime(), nullable=True),
+                    sa.Column('modified_at', sa.DateTime(), nullable=True),
+                    sa.Column('diagnosis_id', dataservice.api.common.model.KfId(
+                        length=11), nullable=False),
+                    sa.Column('biospecimen_id', dataservice.api.common.model.KfId(
+                        length=11), nullable=False),
+                    sa.Column('kf_id', dataservice.api.common.model.KfId(
+                        length=11), nullable=False),
+                    sa.ForeignKeyConstraint(['biospecimen_id'], [
+                                            'biospecimen.kf_id'], ),
+                    sa.ForeignKeyConstraint(
+                        ['diagnosis_id'], ['diagnosis.kf_id'], ),
+                    sa.PrimaryKeyConstraint('kf_id'),
+                    sa.UniqueConstraint('diagnosis_id', 'biospecimen_id'),
+                    sa.UniqueConstraint('uuid')
+                    )
+    op.drop_constraint('diagnosis_biospecimen_id_fkey',
+                       'diagnosis', type_='foreignkey')
+    op.drop_column('diagnosis', 'biospecimen_id')
```

Comment:
We have to do a data migration similar to how we did bs-gf migration.

Reply:
we don't have associated data between biospecimen and diagnosis. do we still need to data migrations?

Reply:
Hmm, as in we haven't added any yet? I still feel that we should add it, just in case. It should be almost identical to the last migration.